internal/cmd/group.go

```diff
@@ -25,8 +25,34 @@ func init() {
 	addCanaryFlag(groupsCreateCmd)
 	groupCmd.AddCommand(groupsDestroyCmd)
 	addYesFlag(groupsDestroyCmd, "Confirms the destruction of the group, with all its locations and databases.")
+	groupCmd.AddCommand(groupShowCmd)
 }
 
+var groupShowCmd = &cobra.Command{
+	Use:               "show <group-name>",
+	Short:             "Show information about a group.",
+	Args:              cobra.ExactArgs(1),
+	ValidArgsFunction: groupArgs,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+		client, err := authedTursoClient()
+		if err != nil {
+			return err
+		}
+
+		name := args[0]
+		group, err := getGroup(client, name)
+		if err != nil {
+			fmt.Printf("group %s not found\n", internal.Emph(name))
```

Comment:
you may just return the error, we should parse any error from the API and display it correctly
printing not found may not be the correct thing in all situations